CMakeLists.txt

```diff
@@ -59,32 +82,97 @@ if (VITA)
-		    FILE platform/vita/sce_sys sce_sys
-		    FILE assets assets
-	)
+     vita_create_vpk(match-theory.vpk ${VITA_TITLEID} match-theory.self
+          VERSION ${VITA_VERSION}
+          NAME ${VITA_APPNAME}
+          FILE platform/vita/sce_sys sce_sys
+          FILE platform/vita/assets/images assets/images
+          FILE platform/vita/assets/backgrounds assets/backgrounds
+          FILE assets/fonts assets/fonts
+     )
 ELSEIF(PLATFORM_PSP)
      set(SDL2_INCLUDE_DIR "${PSPDEV}/psp/include/SDL2")
 
      target_include_directories(match-theory PRIVATE
           ${SDL2_INCLUDE_DIR}
      )
 
-     target_link_libraries(match-theory PRIVATE SDL2_image SDL2 png jpeg z GLU GL pspvram pspvfpu psphprm pspdisplay pspgu pspge pspdebug pspaudio)
+     target_link_libraries(match-theory PRIVATE
+          SDL2_image
+          SDL2_mixer
+          SDL2main
+          SDL2
+          SDL2_ttf
+          png
+          jpeg
+          pspvram
+          GL
+          pspvram
+          pspvfpu
+          pspgu
+          pspctrl
+          pspdebug
+          pspdisplay
+          pspge
+          pspaudio
+          psphprm
+          psprtc
+          jsoncpp
+          vorbisfile
+          vorbis
+          ogg
+          freetype
+          z
+     )
 
      create_pbp_file(
           TARGET match-theory
           ICON_PATH NULL
           BACKGROUND_PATH NULL
           PREVIEW_PATH NULL
           TITLE Match-Theory
+          ENC
           BUILD_PRX
      )
 
+     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/assets)
+     file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/assets/fonts
+          DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/assets)
+     file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/platform/psp/assets/images
+          DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/assets)
+     file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/platform/psp/assets/backgrounds
+          DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/assets)
+
+ELSEIF(WIN32)
+     find_package(SDL2 CONFIG REQUIRED)
+     find_package(sdl2-image CONFIG REQUIRED)
+     find_package(sdl2-ttf CONFIG REQUIRED)
+     find_package(sdl2-mixer CONFIG REQUIRED)
+     find_package(jsoncpp CONFIG REQUIRED)
+
+     target_link_libraries(match-theory PRIVATE
+          SDL2::SDL2
+          SDL2::SDL2main
+          SDL2::SDL2_image
+          SDL2::SDL2_ttf
+          SDL2::SDL2_mixer
+          jsoncpp_lib
+          jsoncpp_object
+          JsonCpp::JsonCpp
+     )
+     target_include_directories(match-theory PRIVATE
+          ${SDL2_INCLUDE_DIRS}
+          ${SDL2_IMAGE_INCLUDE_DIRS}
+          ${SDL2_TTF_INCLUDE_DIRS}
+          ${SDL2_MIXER_INCLUDE_DIRS}
+          ${JSONCPP_INCLUDE_DIRS}
+     )
+     if(NOT CMAKE_BUILD_TYPE MATCHES "Release")
+         set(CMAKE_BUILD_TYPE "Debug")
+     endif()
+     
      file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/assets
-               DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+          DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/)
 ELSE()
```

Comment:
`ELSE()` and `ENDIF()` probably should be written in lowercase for consistency, unless there is a difference between these two that I don't understand.